src/ansible_creator/resources/collection_project/docs/docsite/links.yml.j2

```diff
@@ -32,16 +32,6 @@ extra_links:
 # Specify communication channels for your collection. We suggest to not specify more
 # than one place for communication per communication tool to avoid confusion.
 communication:
-  matrix_rooms:
-    - topic: General usage and support questions
-      room: "#users:ansible.im"
-  irc_channels:
-    - topic: General usage and support questions
-      network: Libera
-      channel: "#ansible"
-  mailing_lists:
-    - topic: Ansible Project List
-      url: https://groups.google.com/g/ansible-project
-      # You can also add a `subscribe` field with an URI that allows to subscribe
-      # to the mailing list. For lists on https://groups.google.com/ a subscribe link is
-      # automatically generated.
+  forum:
+    - topic: Ansible Creator project-related conversations
+      url: https://forum.ansible.com/tag/ansible-creator
```

Comment:
I think this file is a template, so should likely point to a more generic place on the forum

Reply:
Do you mean something like now? PTAL